robottelo/decorators/host.py

```diff
@@ -0,0 +1,120 @@
+import logging
+from functools import wraps
+
+import re
+import six
+import unittest2
+
+from robottelo import ssh
+from robottelo.config import settings
+
+if six.PY3:
+    from functools import lru_cache
+elif six.PY2:
+    # workaround only for this module, its not doing exactly the same as
+    # original from Python 3, but it fits this specific case
+    def lru_cache(maxsize):
+        def decorator(func):
+            cache = []
+
+            @wraps(func)
+            def wrapper():
+                if not cache:
+                    cache.append(func())
+                return cache[0]
+
+            return wrapper
+
+        return decorator
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Let's use _LOGGER now, but we need to standardize the name as we discussed on IRC. Pick any one as @rochacbruno mentioned in previous comment.